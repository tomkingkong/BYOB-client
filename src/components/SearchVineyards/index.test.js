import React from 'react';
import { shallow } from 'enzyme';
import { SearchVineyards } from '.';

describe('SearchVineyards', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<SearchVineyards />);
    expect(wrapper).toMatchSnapshot();
  })
})