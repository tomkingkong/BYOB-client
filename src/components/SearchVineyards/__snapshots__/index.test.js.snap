// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchVineyards should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SearchVineyards />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <input
        className="Search"
        name="search"
        onChange={[Function]}
        placeholder="Search for Vineyard"
        type="text"
        value=""
      />,
      "className": "SearchForm",
      "onKeyUp": [Function],
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "className": "Search",
        "name": "search",
        "onChange": [Function],
        "placeholder": "Search for Vineyard",
        "type": "text",
        "value": "",
      },
      "ref": null,
      "rendered": null,
      "type": "input",
    },
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <input
          className="Search"
          name="search"
          onChange={[Function]}
          placeholder="Search for Vineyard"
          type="text"
          value=""
        />,
        "className": "SearchForm",
        "onKeyUp": [Function],
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "Search",
          "name": "search",
          "onChange": [Function],
          "placeholder": "Search for Vineyard",
          "type": "text",
          "value": "",
        },
        "ref": null,
        "rendered": null,
        "type": "input",
      },
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
